bin/install.sh

```diff
@@ -179,7 +183,12 @@ hasproj() {
 
 export JULIA_PROJECT="@."
 
-cd "${CIRRUS_WORKING_DIR}"
+if [ -z "\${JULIA_PROJECT_SUBDIR}" ]; then
+    cd "${CIRRUS_WORKING_DIR}"
+else
+    cd "${CIRRUS_WORKING_DIR}/${JULIA_PROJECT_SUBDIR}"
```

Comment:
You have `$` escaped in the conditional but not where the variable is used. Is that intentional?

Reply:
I remember that we can escape `$` or not. Both implementations are working.

Reply:
Ah yeah, it makes sense that they'd both work, it's just a matter of resolving the environment variables at the time the script is generated or when it's run, and either way all relevant variables have been defined at that point. 👍